src/tasks/minions/farmingActivity.ts

```diff
@@ -167,8 +168,17 @@ export const farmingTask: MinionTask = {
 			}
 			checkHealthXp = alivePlants * plantToHarvest.checkXp;
 
+			const cleanHerb =
```

Comment:
`cleanHerb` is ambiguous, booleans should almost always start with something like is/was/has/should/did, `shouldCleanHerb` sounds good here